app/models/route.rb

```diff
@@ -0,0 +1,12 @@
+class Route < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  validates :source, :destination, presence: true, format: { with: /\A[A-Za-z\s]+\z/ }
+  attr_accessor :name
```

Comment:
why do we need this?

Reply:
Apu, `attr_accessor` is a shortcut method when we need both `attr_reader` and `attr_writer`.

Reply:
@hoh-wtag  can you please let us know why we are using it here?

Reply:
bhaia, the `name` attribute is being assigned a value in the `full_name` method. However, without the `attr_accessor` line, there wouldn't be a way to access the `name` attribute from outside the class. By using `attr_accessor :name`, the class allows external code to read and modify the `name` attribute using the name getter and setter methods, respectively.

Reply:
Bhaia, removed `attr_accessor`. Thanks for the feedback.